frmts/netcdf/netcdfdataset.h

```diff
@@ -752,6 +757,7 @@ class netCDFDataset final: public GDALPamDataset
     int          nYDimID;
     bool         bIsProjected;
     bool         bIsGeographic;
+    bool         bAssumedLongLat; 
```

Comment:
no need to make that a member variable. This can be local to where it is used

Reply:
cool thank you